src/main/resources/view/HelpWindow.fxml

```diff
@@ -24,7 +24,7 @@
     <scene>
         <Scene>
             <stylesheets>
-                <URL value="@PawbookTheme.css" />
+                <URL value="@DarkTheme.css" />
```

Comment:
The name of the css file has been changed if i'm not mistaken, changing it will break the current theme.

Reply:
Done!